campaigns.json

```diff
@@ -2777,16 +2813,25 @@
     "go": true,
     "lumisize": -1,
     "maxcopies": 1,
-    "SecondaryLocation": [
-       "T1_ES_PIC_Disk",
-       "T1_US_FNAL_Disk"
-    ],
     "SiteBlacklist": [
        "T2_CH_CERN",
        "T2_CH_CERN_HLT"
     ],
-    "secondaries": {
-      "/Neutrino_E-10_gun/RunIIFall17FSPrePremix-PUFSUL16_106X_mcRun2_asymptotic_v16-v1/PREMIX" : {}
+    "secondaries": {    
+      "/Neutrino_E-10_gun/RunIIFall17FSPrePremix-PUFSUL16_106X_mcRun2_asymptotic_v16-v1/PREMIX" :
+                {
+                  "SecondaryLocation": [
+                     "T1_ES_PIC_Disk",
+                     "T1_US_FNAL_Disk"
+                       ]
+                 },
+       "/Neutrino_E-10_gun/RunIIFall17FSPrePremix-PUFSUL16CP5_106X_mcRun2_asymptotic_v16-v1/PREMIX" :
+                {
+                  "SecondaryLocation": [
+                     "T1_RU_JINR_DISK",
```

Comment:
@scarletnorberg `T1_RU_JINR_DISK` should be `T1_RU_JINR_Disk`

Reply:
Ok changed.